lms/static/js/ccx/schedule.js

```diff
@@ -312,10 +349,25 @@ var edx = edx || {};
 		.data('what', what)
 		.data('location', row.data('location'));
 	      modal.find('h2').text(
-		  what === 'due' ? gettext("Enter Due Date") : 
+		  what === 'due' ? gettext("Enter Due Date") :
```

Comment:
This modal allows you to set the time too.  Consider changing to Enter Due Date and Time